support-payment-api/src/test/scala/services/PaypalServiceSpec.scala

```diff
@@ -112,14 +112,14 @@ class PaypalServiceSpec extends AnyFlatSpec with Matchers with MockitoSugar with
   }
 
   it should "return an error if the payment amount exceeds Australia max" in new PaypalServiceTestFixture {
-    val createPaypalPaymentData = CreatePaypalPaymentData(Currency.AUD, 16000.50, "url", "url")
+    val createPaypalPaymentData = CreatePaypalPaymentData(Currency.AUD, 16640.50, "url", "url")
```

Comment:
AU& UK the only countries with upper limits?

Reply:
The existing test only covers AUD and non-AUD for some reason